example/lib/home_page.dart

```diff
@@ -19,7 +19,9 @@ class _MyHomePageState extends State<MyHomePage> {
   void _incrementCounter() => setState(() => _counter++);
 
   void changeTheme() =>
-      widget.themeNotifier.value = widget.themeNotifier.value == ThemeMode.light ? ThemeMode.dark : ThemeMode.light;
+      widget.themeNotifier.value = widget.themeNotifier.value == ThemeMode.light
```

Comment:
I generally use line-length of 120 as a standard for most of the stuff, It would be good if you follow the same for libadwaita.

Reply:
I have nothing against using 120 instead of 80 but it goes against the default dartfm settings. IMO using 120 makes contributing to this project trickier for possible contributors if they have to change their settings just for this package. It also sets this projects apart from others dart packages increasing the complexity of contributing.

I would either advocate for sticking with the usual 80 characters or find a way to set this setting locally with a config file local to this repo, so that possible contributors do not have to change their settings.

Reply:
Ok, you have a valid point.